### PR TITLE
added example for deploying ubuntu 2004 packer machine image #92

### PR DESCRIPTION
**Pull request by Cbkhare**
## Description

This PR adds an example of how to create and deploy packer machine image using Tinkerbell. 

## How Has This Been Tested?
- Vagrant setup with Virtual box provider.

### CODE REVIEW

File: docs/deploying-operating-systems/examples-ubuntu-packer.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+---
+title: Examples - Deploying Ubuntu from Packer Machine image
+date: 2021-04-02T00:00:00.000Z
+---
+
+# Deploying Ubuntu from Packer Machine image
+
+This guide will walk you through how you create a minimalistic raw Ubuntu image using the Packer which can be deployed on a bare metal server using the Tinkerbell.
```
Comment (markyjackson-taulia) [inline docs/deploying-operating-systems/examples-ubuntu-packer.md:8]: I feel like this `using the Tinkerbell.` should read `using Tinkerbell.`

File: docs/deploying-operating-systems/examples-ubuntu-packer.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+## Generating a raw image using Packer
+
+[Packer](https://www.packer.io/) is an awesome tool to build automated machine images. Currently, the Packer doesnot officially provide a way to make bare metal machine images. So in this example, we will use `virtualbox-iso` builder to create a vmdk and then convert it to a raw image. Below, will be the directory structure of the Packer image builder.
```
Comment (markyjackson-taulia) [inline docs/deploying-operating-systems/examples-ubuntu-packer.md:12]: `doesnot` should be `does not`

Comment (markyjackson-taulia) [inline docs/deploying-operating-systems/examples-ubuntu-packer.md:12]: `So in this example` should be `So, in this example`

**Review by markyjackson-taulia — CHANGES_REQUESTED** [review]
Minor grammer changes

File: docs/deploying-operating-systems/examples-ubuntu-packer.md (hunk 2 above)
Comment (Cbkhare) [reply docs/deploying-operating-systems/examples-ubuntu-packer.md]: done

Comment (Cbkhare) [reply docs/deploying-operating-systems/examples-ubuntu-packer.md]: done

File: docs/deploying-operating-systems/examples-ubuntu-packer.md (hunk 1 above)
Comment (Cbkhare) [reply docs/deploying-operating-systems/examples-ubuntu-packer.md]: done

**Review by markyjackson-taulia — APPROVED** [review]

**Review by DailyAlice — APPROVED** [review]
I'm going to go ahead and approve; thinking I would just do the same as the other OS docs, merge and then PR with a couple of markdown fixes and adding it to the toc (mkdocs.yml).

**Pull request merged**